### ISSUE TITLE
no bottle preference menu ?

### ISSUE THREAD

**Issue by Lucaspec72** [auntiepickle/ff7seventhheavensteamdeck#2]
I'm following the guide to set it up, but i'm stuck at "Configuring 7th Heaven" since it requires to check "Improve Direct 3D 9/10/11 Compability", which is in the bottle preference menu which i don't seem to have on my version of bottles.
Idk if it helps, but below are some screenshots of where i currently am.
(i already checked the cog button next to run executable, it's just for launch options)

![image](https://user-images.githubusercontent.com/48445530/202473766-36479cab-9bd5-435d-be5a-614bb87d85df.png)
![image](https://user-images.githubusercontent.com/48445530/202473674-d3ec749c-5d2a-4852-b591-6fd75705ac49.png)
![image](https://user-images.githubusercontent.com/48445530/202473951-47d8eeb6-62d0-4087-8696-25bc8dcc8254.png)

edit: also for some reason i can't seem to be able to restart 7th heaven from the start button on the second image, trying to run it in console gives me this :
![image](https://user-images.githubusercontent.com/48445530/202484145-f86ed080-848a-4e16-9466-41e054089316.png)

**Comment by auntiepickle** [comment]
Looks like there was a UI change in the latest patch! I’ll fix the GitHub page soon but you can disable it by ensuring that dxvk and vkd3d are set to disabled.
________________________________
From: Lucaspec72 ***@***.***>
Sent: Thursday, November 17, 2022 6:35:23 AM
To: auntiepickle/ff7seventhheavensteamdeck ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [auntiepickle/ff7seventhheavensteamdeck] no bottle preference menu ? (Issue #2)


I'm following the guide to set it up, but i'm stuck at "Configuring 7th Heaven" since it requires to check "Improve Direct 3D 9/10/11 Compability", which is in the bottle preference menu which i don't seem to have on my version of bottles.
Idk if it helps, but below are some screenshots of where i currently am.
(i already checked the cog button next to run executable, it's just for launch options)

[image]<https://user-images.githubusercontent.com/48445530/202473766-36479cab-9bd5-435d-be5a-614bb87d85df.png>
[image]<https://user-images.githubusercontent.com/48445530/202473674-d3ec749c-5d2a-4852-b591-6fd75705ac49.png>
[image]<https://user-images.githubusercontent.com/48445530/202473951-47d8eeb6-62d0-4087-8696-25bc8dcc8254.png>

**Comment by Lucaspec72** [comment]
sorry, this is gonna sound dumb, but how exactly do i do that ? (what am i supposed to disable btw, i thought i had to enable DX3D compability mode)

Also, for some reason 7th heaven just refuses to start no matter what i do, i've tried starting it from the bottles app, and also from starting the EXE with bottles from the file manager, nothing seems to work. (even though it did start at the end of the installation, albeit with the black window bug.

**Comment by auntiepickle** [comment]
No worries! Not sure which step you’re at but to disable D3D you change dxvk from a version number to disabled by clicking it.

Also sounds like it might be a runner issue. Sometimes rebooting my steam deck helped but if you screenshot your current settings again I can take a look since it should launch reliably with the right config.
________________________________

**Comment by Lucaspec72** [comment]
yeah i just tried starting 7th heaven with both DXVK and VKD3D disabled, and it loaded just fine, and i can actually see the windows. gonna continue following the guide to see if i can get everything up and running.

**Comment by Lucaspec72** [comment]
well, didn't make it far, for some reason 7th heaven can't find the location of the FF7 folder or something, even though i think i set it right.
![image](https://user-images.githubusercontent.com/48445530/202496613-78bec899-fdac-4178-8e61-8030332ec246.png)
![image](https://user-images.githubusercontent.com/48445530/202496733-b9758000-c675-457a-8695-cd80571bd167.png)

**Comment by auntiepickle** [comment]
Ok! Looks like the ff7.exe is missing. I would check that it does exist in that folder from dolphin and if you still get issues, you can uninstall and reinstall seventh heaven in the bottle.


________________________________

**Comment by Lucaspec72** [comment]
i checked and it's there. also tried deleting it and recreating it (by selecting ff7_en) since maybe it had something to do with the fact i transfered the files from another PC, but this doesn't change a thing.

**Comment by auntiepickle** [comment]
I ran into this issue once and sadly the only way to fix it was recreating the bottle since seventh heaven needs to copy the ff7 file in my experience
________________________________

**Comment by Lucaspec72** [comment]
i made another bottle and set up everything again, same problem...
![image](https://user-images.githubusercontent.com/48445530/202537347-17d7ad63-abd2-45f4-9475-c8b2db8c4c53.png)

**Comment by auntiepickle** [comment]
When you point it at the paths do you initially point it at the ff7_en file and then it copies the ff7 file? It seems like for whatever reason the file isn’t there.
________________________________

**Comment by auntiepickle** [comment]
It could also be a flat seal issue if it doesn't have access to your steam folder
________________________________

**Comment by Lucaspec72** [comment]
this time i selected the ff7.exe directly, and i think flatseal is set up correctly
![image](https://user-images.githubusercontent.com/48445530/202578634-d9df27e5-8507-40de-8600-e52e7aa3ce63.png)

**Comment by Colonelcrabcake** [comment]
I ran into this issue. While snooping around I realized maybe I screwed up a step.
In the FVII_STEAM_FOLDER there was a symlink I think to FINAL_FANTASY_7 or something like that but that folder just linked to the parent folder, so it was a endless directory.
I deleted that folder and it booted right up.
I don't have screenshots but if you still have trouble I will make a new bottle. Maybe we can get to the bottom of this.
